app/decorators/data_request_area_decorator.rb

```diff
@@ -22,7 +22,7 @@ def location
   end
 
   def request_document
-    "#{data_request_area_type.humanize} records"
+    commissioning_document.decorate.request_document
```

Comment:
Not sure whether its best to use it through commissioning_document rather specifically using a translation. It makes some of the tests require `let(:commissioning_document) { create :commissioning_document, data_request_area: }` which seems messy

happy to change